<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>get_date_form</class>
 <widget class="QWidget" name="get_date_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Укажите дату</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>Укажите месяц и год отчета</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="month_combo_box"/>
   </item>
   <item>
    <widget class="QSpinBox" name="year_spin_box"/>
   </item>
   <item>
    <widget class="QPushButton" name="create_report_btn">
     <property name="text">
      <string>Создать отчет</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
